client/signup/steps/domains/index.jsx

```diff
@@ -83,7 +83,13 @@ import { getExternalBackUrl, shouldUseMultipleDomainsInCart } from './utils';
 import './style.scss';
 
 const BoldTLD = ( { domain } ) => {
-	const tld = domain.split( '.' ).pop();
+	const split = domain.split( '.' );
+	let tld = split.pop();
+	const wp = split.pop();
+	if ( wp === 'wordpress' && tld === 'com' ) {
+		tld = `wordpress.com`;
+	}
+
```

Comment:
Pretty minor, but wondering if we need to add this for the other free subdomains we support, I found a list of it in this message: p1665453210588499/1665427916.681429-slack-C0761SX4K

Reply:
Oh, interesting! I don't _think_ we need to add support for those subdomains in this PR. We're checking for wordpress.com in several other places in this component, so we'd probably need several updates to support those subdomains. I'm going to merge this PR, but we can follow up if needed.